### PR TITLE
Add xyz.dimtpap.coppwr

### PR DESCRIPTION
**Pull request by dimtpap**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by dimtpap** [comment]
bot, build xyz.dimtpap.coppwr

File: xyz.dimtpap.coppwr.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+		"--share=ipc",
+		"--filesystem=xdg-run/pipewire-0"
+	],
+	"build-options": {
+		"append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin",
+		"prepend-ld-library-path": "/usr/lib/sdk/llvm16/lib"
+	},
+	"modules": [
+		{
+			"name": "pipewire",
```
Comment (hfiguiere) [inline xyz.dimtpap.coppwr.json:24]: pipewire is in the runtime.

File: xyz.dimtpap.coppwr.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+				"install -Dm644 LICENSE /app/share/licenses/coppwr/LICENSE",
+				"install -Dm644 assets/xyz.dimtpap.coppwr.desktop /app/share/applications/xyz.dimtpap.coppwr.desktop",
+				"install -Dm644 assets/xyz.dimtpap.coppwr.metainfo.xml /app/share/metainfo/xyz.dimtpap.coppwr.metainfo.xml",
+				"install -Dm644 assets/icon/scalable.svg /app/share/icons/hicolor/scalable/apps/xyz.dimtpap.coppwr.svg",
+				"install -Dm644 assets/icon/512.png /app/share/icons/hicolor/512x512/apps/xyz.dimtpap.coppwr.png",
+				"install -Dm644 assets/icon/256.png /app/share/icons/hicolor/256x256/apps/xyz.dimtpap.coppwr.png",
+				"install -Dm644 assets/icon/128.png /app/share/icons/hicolor/128x128/apps/xyz.dimtpap.coppwr.png",
+				"install -Dm644 assets/icon/64.png /app/share/icons/hicolor/64x64/apps/xyz.dimtpap.coppwr.png",
+				"install -Dm644 assets/icon/48.png /app/share/icons/hicolor/48x48/apps/xyz.dimtpap.coppwr.png",
+				"install -Dm644 assets/icon/32.png /app/share/icons/hicolor/32x32/apps/xyz.dimtpap.coppwr.png"
```
Comment (barthalion) [inline xyz.dimtpap.coppwr.json:78]: SVG is enough, if other files are not used by the app directly.

**Comment by barthalion** [comment]
bot, build xyz.dimtpap.coppwr

**Pull request closed without merging**